lib/hydra/derivatives/image.rb

```diff
@@ -33,7 +34,7 @@ def new_mime_type(format)
 
       def create_resized_image(output_file, size, format, quality=nil)
         create_image(output_file, format, quality) do |xfrm|
-          xfrm.resize(size) if size.present?
+          xfrm.scale(size) if size.present?
```

Comment:
I'd say revert this change unless it substantially helps you deliver RAW support in this PR.